javaagent-bootstrap/src/test/java/io/opentelemetry/javaagent/bootstrap/ConfiguredResourceAttributesHolderTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.bootstrap;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.common.Attributes;
+import java.util.Collections;
+import org.junit.jupiter.api.Test;
+import org.junitpioneer.jupiter.ClearSystemProperty;
+import org.junitpioneer.jupiter.SetSystemProperty;
+
+class ConfiguredResourceAttributesHolderTest {
+
+  @Test
+  @SetSystemProperty(
+      key = "otel.instrumentation.mdc.resource-attributes",
+      value = "service.name,runtime")
+  void testGetAttributeValue() {
+    Attributes attributes = Attributes.builder().put("service.name", "test-service").build();
+
+    ConfiguredResourceAttributesHolder.initialize(attributes);
+    assertEquals(
```

Comment:
Nit: I think the preference across the repo is to favor assertJ `assertThat(whatever).isEqualTo(otherThing)`.